index.tsx

```diff
@@ -58,10 +59,10 @@ function normalizeBindingNames<Props>(
     return fromPairs(bindingNames.map(_ => [_, '<'])) as AngularBindings<Props>
   }
   if (bindingNames) {
-    return mapValues(bindingNames, _ => _.optional ? '<?' : '<')
+    return mapValues(bindingNames, (_: {optional: boolean}) => _.optional ? '<?' : '<') as AngularBindings<Props>
```

Comment:
Do you really need to manually type `_`?

Reply:
I've been weird different typing issues as though my typescript version is different or something. If you remove this and it compiles for you I'm not at all attached to this.